### DISCUSSION TITLE
INC/DEC preset for button

### DISCUSSION THREAD

**Discussion by willmodelisme** [shanteacontrols/OpenDeck#244] [General]
Hello. Is there a way to set a button to INC/DEC preset with the preset number displayed to the I2C OLED ?

**Comment by paradajz** [comment]
I'm not sure what do you mean.

**Comment by willmodelisme** [comment]
The basic idea is the set a button that will increment preset by 1.
The active preset number will be shown to the I2C OLED.
It will cycle from 0 to 9.
Even better, 2 buttons, one is incrementing (+1 each press) and another could decreasing (-1 each press) the preset number.

**Comment by paradajz** [comment]
It's on the roadmap. When - I don't know. I have quite limited time for new features these days.

**Comment by willmodelisme** [comment]
No pb, tks for your answer.

**Comment by paradajz** [comment]
The preset number is now displayed on display.

**Comment by willmodelisme** [comment]
Hey what’s up ?
Happy to hear that .
Will try tomorrow.
Thank you for the mail.

William.
Envoyé de mon iPhone

Le 13 mars 2024 à 22:42, paradajz ***@***.***> a écrit :

﻿

The preset number is now displayed on display.
